app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
I agree that duplication is OK in some cases, but I don't think this is one of them. We do have a clear abstraction here. Our ultimate goal here is to make it easy for developers to define which attributes should be encrypted by specifying them in the model. For example, the `attr_encrypted` gem allows you to do it like this in any model:
```ruby
class User
  attr_encrypted :ssn
  attr_encrypted :email
  attr_encrypted :password
end
```
In #1070, Peter is allowing the attributes to be defined like this:
```ruby
encrypted_attribute(name: :phone, default: nil)
encrypted_attribute(name: :otp_secret_key, default: nil)
encrypted_attribute(name: :email, default: '', setter: false)
```
To achieve this ease of use, some metaprogramming is required, which I think is totally worth it to be able to easily add or remove encrypted attributes. This makes the app easy to extend and change.